config/database.yml

```diff
@@ -2,7 +2,7 @@ default: &default
   adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
-  user: postgres
+  username: postgres
```

Comment:
That's what I mean XD

Actually, take a look at this StackOverflow question XD

https://stackoverflow.com/questions/33470898/rails-gitignore-still-tracking-my-database-yml-file